.github/workflows/generate-checksums.yml

```diff
@@ -181,10 +183,10 @@ jobs:
           name: ${{ needs.generate-checksum.outputs.artifact-name }}
           path: ${{ env.OUTPUT_LOCAL_LOCATION }}
 
-      - name: Move Checksums to Repo
+      - name: Move Outputs to Repo
         run: |
           mkdir -p ${{ inputs.committed-checksum-location }}
-          mv ${{ env.OUTPUT_LOCAL_LOCATION }}/checksum/* ${{ inputs.committed-checksum-location }}
+          mv ${{ env.OUTPUT_LOCAL_LOCATION }} .
 
```

Comment:
I think you either need to `cd` into `${{ inputs.committed-checksum-location }}` or `cp` into that location. No?

Reply:
I realised that we don't actually need that input anymore - `testing/checksum` is assumed in a lot of places. Removed in https://github.com/ACCESS-NRI/model-config-tests/pull/74/commits/539f893e3cc52c404e5a31d3745f7b3bd042b1fe